packages/platform-ios/src/commands/runIOS/index.ts

```diff
@@ -128,7 +128,17 @@ async function runOnSimulator(
     throw new CLIError('Could not parse the simulator list output', error);
   }
 
-  const selectedSimulator = findMatchingSimulator(simulators, args.simulator);
+  /**
+   * If provided simulator does not exist, try simulators in following order
+   * - iPhone X
+   * - iPhone 8
+   */
+
+  const fallbackSimulators = ['iPhone X', 'iPhone 8'];
+  const selectedSimulator = fallbackSimulators.reduce((simulator, fallback) => {
+    return simulator || findMatchingSimulator(simulators, fallback);
+  }, findMatchingSimulator(simulators, args.simulator));
```

Comment:
This loop always run through all simulators from `fallbackSimulators`, but since the data set is small and we avoid running `findMatchingSimulator` if necessary, I'm fine with that.